New Data Source: azurerm_subscription_locations (#15181)
hashicorp/terraform-provider-azurerm: website/docs/d/extended_locations.html.markdown

---
subcategory: "Base"
layout: "azurerm"
page_title: "Azure Resource Manager: azurerm_extended_locations"
description: |-

>   Get information about the available extended locations per the Azure location for the Subscription ID of the Provider.

Reviewer: we should make this:

```suggestion
  This data source return the available Extended Locations for a specific Azure Region.
```

---

# Data Source: azurerm_extended_locations

> Use this data source to access information about the available extended locations per the Azure Location for the Subscription ID of the Provider.

Reviewer: we should make this:

```suggestion
This data source return the available Extended Locations for a specific Azure Region.
```

## Example Usage

```hcl

> data "azurerm_client_config" "current" {}
> 

Reviewer: we're not using the client config in this, so we can remove it?

```suggestion
```

data "azurerm_extended_locations" "example" {
  location = "West Europe"
}
```

## Argument Reference

> * `location` - The Azure location that the Extended Location corresponds.
> 
> ## Attributes Reference
> 
> * `id` - The ID of the Extended Locations.
> 
> * `extended_locations` - The available extended locations for the Azure Location.

Reviewer: we should make these:

```suggestion
* `location` - The Azure location to retrieve the Extended Locations for.

## Attributes Reference

* `id` - The ID of Location within this Subscription.

* `extended_locations` - The available extended locations for the Azure Location.
```
